src/solver/bounded.rs

```diff
@@ -0,0 +1,1429 @@
+// Copyright 2022-2023 VMware, Inc.
+// SPDX-License-Identifier: BSD-2-Clause
+
+//! A bounded state interpreter over a set-based message-passing language.
```

Comment:
I don't really know what this means. If you have a reference you could just link to it, otherwise I'd like to have at least a brief explanation of what "bounded state" and "set-based message-passing language" mean. (It's okay if that explanation is too terse for me for now but let's at least put a paragraph in.)

Reply:
Whoops, that's out-of-date.